### PR TITLE
Master es

### PR DESCRIPTION
**Pull request by WalterC87**
create the translte to spanish for PHP file

### CODE REVIEW

File: angularjs.html.markdown [hunk 1]
```diff
@@ -0,0 +1,709 @@
+---
+language: AngularJS/JavaScript
+contributors:
+    - ["Walter Cordero", "http://waltercordero.com"]
+translators:
+    - ["Walter Cordero", "http://waltercordero.com"]     
+filename: learnangular.html
+---
+
+Feedback is welcome! You can find me in:
```
Comment (vendethiel) [inline angularjs.html.markdown]: @adambard should we cleanup tutorials from this? We already have the contributor link at the top

File: angularjs.html.markdown [hunk 2]
```diff
@@ -0,0 +1,709 @@
+---
+language: AngularJS/JavaScript
+contributors:
+    - ["Walter Cordero", "http://waltercordero.com"]
+translators:
+    - ["Walter Cordero", "http://waltercordero.com"]     
+filename: learnangular.html
```
Comment (vendethiel) [inline angularjs.html.markdown]: it needs the tool tag

File: angularjs.html.markdown (hunk 1 above)
Comment (vendethiel) [inline angularjs.html.markdown]: @adambard ping please

Comment (adambard) [inline angularjs.html.markdown]: Yeah, probably should go. It's not a translation if you wrote it in that language initially.

File: angularjs.html.markdown (hunk 2 above)
Comment (vendethiel) [inline angularjs.html.markdown]: @adambard should we have a Framework tag?

**Comment by vendethiel** [comment]
@adambard ping :)

**Comment by adambard** [comment]
I merged in the angular one; the php we already have in there (and I'm not qualified to tell which is better) and the html5 articles are written using \/\/ comments throughout the code, which is just confusing.

**Pull request closed without merging**